yarn.lock

```diff
@@ -11590,6 +11346,13 @@ ts-api-utils@^1.3.0:
   resolved "https://registry.yarnpkg.com/ts-api-utils/-/ts-api-utils-1.3.0.tgz#4b490e27129f1e8e686b45cc4ab63714dc60eea1"
   integrity sha512-UQMIo7pb8WRomKR1/+MFVLTroIvDVtMX3K6OUir8ynLyzB8Jeriont2bTAtmNPa1ekAgN7YPDyf6V+ygrdU+eQ==
 
+ts-blank-space@^0.4.1:
+  version "0.4.1"
+  resolved "https://registry.yarnpkg.com/ts-blank-space/-/ts-blank-space-0.4.1.tgz#3ddc6c7c4009337ffb1811071a8fc55d8076b619"
+  integrity sha512-i0JvTvFaZcXCj+tcs67G0K3zXgUr1ebGvhmuB3ov0ByFrbuvTOgUwtFEOP8dN+B+ou0wkhosJ8Q4RPfi8mHFQQ==
+  dependencies:
+    typescript "5.1.6 - 5.6.x"
```

Comment:
This might become interesting when we try to upgrade tsc and blank-space hasn't caught up.